templates/gift_card.liquid

```diff
@@ -49,6 +49,9 @@
         --font-heading-style: {{ settings.type_header_font.style }};
         --font-heading-weight: {{ settings.type_header_font.weight }};
 
+        --font-body-scale: {{ settings.body_scale | divided_by: 100.0 }};
+        --font-heading-scale: {{ settings.heading_scale | times: 1.0 | divided_by: settings.body_scale }};
```

Comment:
I am curious as why you are keeping the decimal instead of using `divided_by: 100` and `times: 1`

Reply:
Dividing and multiplying integers using Liquid filter produces an integer result, but for scale, we need decimal values